library/src/lib/localizator-editor/localization-editor.component.scss

```diff
@@ -0,0 +1,5 @@
+::ng-deep {
```

Comment:
Instead of making use of ::ng-deep, **remove style encapsulation** from the components